### PR TITLE
feature: Add loading indicator

### PR DESCRIPTION
**Pull request by Jchann24**
# Add Loading Indicator

Reference : #10 

I've managed to added the loading indicator to the `CheckRepository.vue` component.
I'm using `vue-spinner`.

Please check my PR!
Thanks 🍻

### CODE REVIEW

File: components/CheckRepository.vue [hunk 1]
```diff
@@ -120,14 +127,24 @@
...
+  },
   data() {
     return {
       url: null,
       processing: false,
       errors: [],
       result: null,
       previous: [],
+      loaderOpt: {
+        color: '#ff8ae2',
```
Comment (lukeocodes) [inline components/CheckRepository.vue:144]:
```suggestion
        lightColor: '#9c4668',
        darkColor: '#ff8ae2',
```

File: components/CheckRepository.vue [hunk 2]
```diff
@@ -72,7 +72,14 @@
               Check another?
             </a>
           </div>
-          <div v-else-if="processing" class="Processing">Processing...</div>
+          <div v-else-if="processing" class="Processing">
+            <scale-loader
+              :loading="processing"
+              :color="loaderOpt.color"
```
Comment (lukeocodes) [inline components/CheckRepository.vue:78]:
```suggestion
              :color="$colorMode.value === 'dark' ? loaderOpt.darkColor : loaderOpt.lightColor"
```

**Review by lukeocodes — CHANGES_REQUESTED** [review]
Can we try these changes out to see if they work for supporting lightmode/darkmode toggle?

File: components/CheckRepository.vue (hunk 2 above)
Comment (Jchann24) [reply components/CheckRepository.vue]: Sorry I forgot, 
I've implement those changes in the latest commit! Thanks!

**Review by lukeocodes — APPROVED** [review]
Really love this loader, you made it look awesome.

**Pull request merged**